sdk/core/azure-core/src/main/java/com/azure/core/util/logging/LoggingUtils.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util.logging;
```

Comment:
Can we move this to implementation if it is only going to contain implementation utility methods.

Reply:
I definitely can, but curious why? It's a package-private class with package-private method and I only moved it away from `ClientLogger` for testing purposes.